src/lgtm/scripts/download/pyroscope.sh

```
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source "$(dirname "$(dirname "$(readlink -f "$0")")")/lib/download/_.sh"
```

Shellcheck (0.10.0)

[info] Not following: ./lib/download/_.sh was not specified as input (see shellcheck -x).

(SC1091)